app/views/series/_series_navigation.html.erb

```diff
@@ -16,4 +8,12 @@
       <li><%= link_to ts("Remove Me As Author"), edit_series_path(@series, :remove => 'me'), data: {confirm: series_removal_confirmation(@series, current_user)} %></li>
     <% end %>
   <% end %>
+  <li><%= span_if_current ts("Series"), series_path(@series) %></li>
+  <% if @subscription %>
+    <li>
+      <%= render :partial => 'subscriptions/form', :locals => { :subscription => @subscription } %>
```

Comment:
Could you update the syntax and switch to double quotes here? `<%= render "subscriptions/form", subscription: @subscription %>`